opencti-platform/opencti-front/src/private/components/settings/decay/DecayRules.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import Alert from '@mui/material/Alert';
+import { buildEntityTypeBasedFilterContext, emptyFilterGroup } from '../../../../utils/filters/filtersUtils';
+import { usePaginationLocalStorage } from '../../../../utils/hooks/useLocalStorage';
+import useQueryLoading from '../../../../utils/hooks/useQueryLoading';
+import CustomizationMenu from '../CustomizationMenu';
+import DecayRuleCreation from './DecayRuleCreation';
+import { DecayRulesLineDummy } from './DecayRulesLine';
+import DecayRulesLines, { decayRulesLinesQuery } from './DecayRulesLines';
+import { useFormatter } from '../../../../components/i18n';
+import ItemBoolean from '../../../../components/ItemBoolean';
+import ListLines from '../../../../components/list_lines/ListLines';
+import type { Theme } from '../../../../components/Theme';
+import { DecayRulesLinesPaginationQuery, DecayRulesLinesPaginationQuery$variables } from './__generated__/DecayRulesLinesPaginationQuery.graphql';
+import { DecayRulesLine_node$data } from './__generated__/DecayRulesLine_node.graphql';
+import useAuth from '../../../../utils/hooks/useAuth';
+import { INDICATOR_DECAY_MANAGER } from '../../../../utils/platformModulesHelper';
+
+const LOCAL_STORAGE_KEY = 'view-decay-rules';
+
+const useStyles = makeStyles<Theme>(() => ({
```

Comment:
`makeStyles` is deprecated. Try to put style directly in JSX using `style={{...}}` for html elements or `sx={{...}}` for MUI elements